### ISSUE TITLE
Use of ethernet connection

### ISSUE THREAD

**Issue by beckerth** [zivillian/heidelberg-1p3p#5]
Hi,

would it be possible to use the wired ethernet connection? 

Cheers,
Tommi

**Comment by beckerth** [comment]
Unfortunately I'm absolutely not an expert in ESP32. But to gather all information which I could obtain up to now:

- according to the board manufacturer it is necessary to "[patch ESP-IDF](https://github.com/dtlzp/relay_dev_demo#step-2-patch-dingtian-code-for-esp-idf)" 
- this project is compiled within Arduino framework ("framework = arduino") and not with "ESP-IDF" framework
- According to another comment I found [here ](https://github.com/espressif/arduino-esp32/discussions/8398#discussioncomment-6537876)  it should be possible to borrow the PHy JL1101 driver from the Tasmota Arduino fork.

**Comment by beckerth** [comment]
Someone who can support here or give some hint how to port the Ethernet driver? ;-)

**Comment by fnsolar** [comment]
I am also interested in a solution, as otherwise I would have to run an extra access point for the box. I prefer a LAN connection wherever possible.

**Comment by jetronic18s** [comment]
@zivillian 
Is there still interest in Ethernet support for the board?

I've been tinkering with the network for a while now and could send you something.

**Comment by fnsolar** [comment]
Yes, I would definitely prefer Ethernet, even though I have now managed to get WiFi working quite stably.

**Comment by jetronic18s** [comment]
I sent a pull request to zivillian.
Let's see how things progress and what else needs to be done.

**Comment by jetronic18s** [comment]
I have saved a version here for you to test:
https://github.com/evcc-io/evcc/discussions/6168#discussioncomment-15758833
